src/annotation.py

```diff
@@ -51,13 +51,17 @@ def annotate_entities(
                     f"What is the label of '{entity['ent']} (pos {entity['start']}:{entity['end']})? In source: {entity['in_source']}"
                 )
                 user_input = ""
-                while user_input not in ["0", "1", "S", "N"]:
```

Comment:
Ah, I guess `N` should have been removed before in 526d446? Either way, all good.